string robot_name       # Robo1
string goal_location    # I1
string task_assignment  # 입고

# 변경예정
#string robot_name      Robo1
#string task_code  "Task_2
#string[] rack_list  ["R_A1", "R_B2", "R_C3"]
#string task_assignment 입고